test/utils/comparable.py

```diff
@@ -146,52 +146,100 @@ def assert_choices_equal(choices1, choices2):
                                                   getattr(choices2, key))
 
 
-def assert_patch_equal(patch1, patch2, deep=False):
-    assert isinstance(patch1, PatchOperation), "patch1 was not an PatchOperation"
-    assert isinstance(patch2, PatchOperation), "patch2 was not an PatchOperation"
-    assert type(patch1) is type(patch2), "patch1 and instance2 are not the same type."
+def default_assert_equal(obj1, obj2, **kwargs):
```

Comment:
Ugg. I did almost the same thing here. Not looking forward to that merge conflict...